fixtures/gists-app/tests/route-data-test.ts

```diff
@@ -0,0 +1,90 @@
+import type { Browser, Page } from "puppeteer";
+import puppeteer from "puppeteer";
+
+import { disableJavaScript, getHtml, reactIsHydrated } from "./utils";
+
+const testPort = 3000;
+const testServer = `http://localhost:${testPort}`;
+
+describe("route data", () => {
+  let browser: Browser;
+  let page: Page;
+  beforeEach(async () => {
+    browser = await puppeteer.launch();
+    page = await browser.newPage();
+  });
+
+  afterEach(() => browser.close());
+
+  describe("the parent should render the child title", () => {
```

Comment:
As @kentcdodds also requested in https://github.com/remix-run/remix/pull/2411#discussion_r830625426

> I think we’re trying to use Ryan’s `createFixture` utility as much as possible instead of the gist app. Could you switch this test to use that instead?

Reply:
Agreed. We are trying to get rid of the gists app for testing stuff and use a separate `createFixture` test instead. Thanks!